<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemToInventory</class>
 <widget class="QWidget" name="AddItemToInventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="TopLayout">
    <item>
     <layout class="QGridLayout" name="InputLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ItemNameLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Item Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ItemNameEntry"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ItemCodeLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Item Code</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ItemCodeEntry"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ItemPriceLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Item Price</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ItemPriceEntry"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ItemQuantity">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Item Quantity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="ItemQuantityEntry"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ButtonsLayout">
      <item>
       <widget class="QPushButton" name="AddItem">
        <property name="text">
         <string>Add To Inventory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
